frontend/src/app/feature/messages/show-more/app-show-more.component.ts

```diff
@@ -86,7 +87,7 @@ export class AppShowMoreComponent
         // observe the height of the content and update the showingMore-state whenever it changes
         // therefore we make use of up to three different strategies to detect changes in the scrollHeight
         if (this.heightChangeObservationStrategies.polling) {
-            setInterval(
+            this.pollingIntervallRef = setInterval(
                 () => this.updateState(),
                 this.heightChangeObservationStrategies.polling
             );
```

Comment:
Doesn't `setInterval` expect a number as its second argument? `this.heightChangeObservationStrategies.polling` is `false`, or not?

Reply:
It was `number | false` (and `0` would have created a performance problem).

I changed it to `boolean`.

Reply:
Not what I meant, but you're change solved my issue.